src/styles/themes/components/_navigationBar.scss

```diff
@@ -0,0 +1,66 @@
+.header {
+  width: 80%;
+  height: 4rem;
+  background-color: $c-background;
+  box-shadow: 0 1px 4px rgba(0, 0, 0, 0.2);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 10%;
+  font-family: $f-baseFontFamily;
+
+  &__logo {
+    display: flex;
+    align-items: center;
+  }
+  &__img {
+    width: 40px;
+  }
+
+  &__title {
+    color: $c-lightBlue;
+    text-transform: uppercase;
+    text-shadow: $c-darkBlue;
+    font-weight: bold;
+  }
+
+  ul {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    align-items: center;
+  }
+
+  li {
+    margin: 0 1rem;
+    cursor: pointer;
+
+    &:hover,
+    &:focus {
+      color: $c-lightBlue;
+    }
+  }
+
+  &__icon {
+    width: 1.5rem;
+    height: 1.5rem;
+    color: $c-greyLight;
+    padding-left: 20px;
+
+    &:hover,
```

Comment:
li:hover header__icon{
   color:  $c-lightBlue;
}

Reply:
i mean somethig like thath
![obraz](https://user-images.githubusercontent.com/56917808/149842609-ce97312e-8500-4c21-86a4-dd2ad3f30d42.png)

Reply:
I remove this and change to material ui navbar